README.md

```diff
@@ -48,3 +48,16 @@ If you would like to see a Swift example, take a look at
 [getting started guides](https://developers.google.com/identity/sign-in/ios/start-integrating).
 * Take a look at the
 [API reference](https://developers.google.com/identity/sign-in/ios/api/).
+
+## App Extensions
+
+Restoring a previous sign-in is supported in [app extensions](https://developer.apple.com/app-extensions/).  The initial
+sign-in flow will need to have been performed in the host app.  In order to allow your app extension to successfully call
+`restorePreviousSignInWithCallback:`, you will need to enable the Keychain Sharing capability and add a shared
```

Comment:
Worth linking to [this doc](https://developer.apple.com/documentation/security/keychain_services/keychain_items/sharing_access_to_keychain_items_among_a_collection_of_apps)? Is this the right doc?

Reply:
Good thought.  Let's revisit in a few weeks.